site/_posts/2021-06-28-identiverse-conference-report.md

```diff
@@ -0,0 +1,90 @@
+* There were a large number of virtual talks, both live and pre-recorded.
+* A number of attendees were virtual. (I don't know how that experience was.)
+
+I'd expect these changes to affect the unit economics of event organizing. Anecdotally, I didn't see anybody checking vaccination status; there was no rapid testing station, for example. I also didn't see many attendees wearing masks, though many of the staff were. I shook hands with a few folks, but there were plenty of hand waves instead. Believe you me, I typically kept social distance and avoided large groups.
+
+The in-person talks were much more fun than the virtual ones. It was easier to ask questions, to interact with fellow attendees, and in general felt more spontaneous. One speaker said, "well, since I'm not being recorded..." and proceeded to give an unvarnished opinion about a dicey matter. While I don't think virtual events are going away, in-person events will come back strong, as the experience is simply higher-bandwidth and more fun.
+
+At the end of the conference, I felt like they'd done a great job handling my covid concerns.
+
+## What about the actual conference, Dan?
+
+As at any conference, there was a plethora of great content and I couldn't attend all the talks. But from what I saw, there were several themes. 
+
+### Identity proofing
+
+First, affordable identity proofing at scale is still an unsolved problem; one that a lot of people are working on. Identity proofing is the process of knowing who someone is before you grant them a user account. It can also be used for account recovery. 
+
+For some applications identity proofing is unimportant; a typical ecommerce site will happily provide an account and take my money without knowing who I actually am. But for many online systems, such as unemployment insurance or business accounts, tying the user to a real identity is really important. There was a talk about how the New York Air National Guard used identity proofing and digital identity products to allow access to sensitive systems for a set of authorized people. As covid forced large numbers of employees toware remote work, knowing folks are who they say they are at scale becomes more critical.
+
+The solutions in this space seem to focus on:
+
+* Automatic recognition of government issued paper documents.
+* Facial recognition, including measures to thwart presentation of photos in place of real people.
+* Ensuring there are manual fallbacks for people when machine learning fails, or when the user has can't present all the required documentation.
+
+This is a step before the authentication and authorization process which is the focus of FusionAuth. While you can identity proof out of band using one of the many solutions out there, if you'd like tighter integration with FusionAuth, I [filed a tracking issue](https://github.com/FusionAuth/fusionauth-issues/issues/1280) which you can upvote.
```

Comment:
Identity Proofing occurs before the authentication and authorization process (which is the focus of FusionAuth).